core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java

```diff
@@ -61,6 +61,13 @@
   @InjectProperty(keys = {"request.${op-any-priority}.timeout", "request.timeout"}, defaultValue = "30000")
   private long msRequestTimeout;
 
+  /**
+   * whether to remove certain headers from the 3rd party invocations
```

Comment:
priority keys in the middle, maybe will cause conflict?
better to move to the tail?

Reply:
I haven't got it why here may cause confict. Since there is always a tail `.clientRequestHeaderFilterEnabled` in this config key, no matter users name their service/schema/operation as "clientRequestHeaderFilterEnabled", the config key seems no problem?

Reply:
fixed